README.md

```diff
@@ -59,31 +65,38 @@ the [kerl](https://github.com/kerl/kerl) build system, and installs them to
 `/usr/local/kerl` for activation before builds. This version is intended for use
 in standard CI runs, such as for pull requests.
 
-# Building a cross-architecture Docker image
+## Building a cross-architecture Docker image
 
-This only works from an `x86_64` build host.
-
-First, configure your machine with the correct dependencies to build multi-arch binaries:
+We can use Docker's
+[Buildx](https://docs.docker.com/buildx/working-with-buildx/) plugin to generate
+multi-architecture container images with a single command invocation. Docker
+Desktop ships with buildx support, but you'll need to create a new builder to
+use it:
 
 ```
-docker run --privileged --rm tonistiigi/binfmt --install all
+docker buildx create --use
```

Comment:
`buildx` doesn't seem to be available in the open source packages (Ubuntu LTS 20.04 for instance) and only from if installed from Docker Desktop, or from the deb package installed from docker.io directly.  Let's leave instruction how to do it without docker.io closed source bits around as well.  For example, when generating the latest package I didn't use buildx and used the instructions which are being deleted in the PR.

Reply:
Hm, OK. Fair request.

One issue is that the previous process only produced arch-specific images and didn't do the work to produce a single multi-architecture manifest. That's fixable.

I'll also have to see about reworking the dockerfiles to allow for the old $CONTAINERARCH approach. Let me see if there's a simple way to do that.

Reply:
Fixed in ed7c197